### ISSUE TITLE
tiny api key required

### ISSUE THREAD

**Issue by JannIds** [dgvirtual/bonfire2-pages-module#2]
![image](https://github.com/dgvirtual/bonfire2-pages-module/assets/126899688/cd4910e9-0f19-41d1-8700-cd9b8e8a43ac)

**Comment by dgvirtual** [comment]
Yes, I guess I need to extract the API key variable as a property in Pages config file and use it in this line in controller: https://github.com/dgvirtual/bonfire2-pages-module/blob/30425a680b24ae1221550998a444a85be7145e2c/Pages/Controllers/PagesController.php#L275

Using API key is avoidable if one uses self-hosted instance of tinyMCE, but that is not very practical.

**Comment by dgvirtual** [comment]
Sort of solved long ago by commit 95a67e3 (moving tinymce api key to Config\Pages)
